contrib/freshmaker-cli

```diff
@@ -224,6 +239,48 @@ def get_freshmaker_events_by_ids(api_url, ids):
     events = sorted(events, key=lambda k: k['id'])
     return events
 
+def get_advisories_from_errata(errata_url):
+    """
+    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED state. Returns the
```

Comment:
```suggestion
    Sends the GET request to the Errata filter for the latest RHSA, in SHIPPED_LIVE state. Returns the
```

Reply:
What does "latest" mean? Is it in the past 24 hours?

Reply:
I had also this question, what does "latest" mean. It's written like this in the JIRA card. 
But as we use the custom filter, which is sorted by newest and this filter shows us only 20 advisories (on the first page which basically we use ) I've decided don't check any dates and check all of this 20 advisories.

Reply:
@mulaievaRH who created the filter? You can ask them and then document it.

Reply:
Never mind. It looks like you documented it. Thanks!